.travis.yml

```diff
@@ -17,14 +16,36 @@ env:
     - DB=MySQLi
     - DB=SQLite
 #    - DB=PostgreSQL
+
 matrix:
+  fast_finish: true
+  include:
+    - php: hhvm
+      sudo: true
+      dist: trusty
+      group: edge # use until the next Travis Image update
+      addons:
+        apt:
+          packages:
+            - mysql-server-5.6
+            - mysql-client-core-5.6
+            - mysql-client-5.6
+      env: DB=MySQLi
+    - php: hhvm
+      sudo: true
+      dist: trusty
+      group: edge # use until the next Travis Image update
+      env: DB=SQLite
   allow_failures:
     - php: nightly
-# Travis have too old HHVM, can't rely on it yet
     - php: hhvm
 #    - env: DB=PostgreSQL
+
 before_script:
-  - mysql -e 'CREATE DATABASE `travis`;'
+  - |
```

Comment:
I created the user, but the related failures were not solved. The following message is in the tests
```php
Access denied for user 'travis'@'localhost' to database 'travis'
```

I tried granting permissions but that also didn't solve the errors, I just got new messages
```php
Warning: Argument 1 not passed as reference in /core/drivers/DB/MySQLi.php on line 118
Fatal error: Call to undefined method mysqli_stmt::get_result() in /core/drivers/DB/MySQLi.php on line 123
```